src/main/java/mx/erickb/shipping/service/RoutesManagerService.java

```diff
@@ -0,0 +1,47 @@
+package mx.erickb.shipping.service;
+
+import mx.erickb.shipping.util.RoutesGraph;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
```

Comment:
Avoid wildcard import, packages like java.util can be really large

Reply:
Agree